pandas/core/base.py

```diff
@@ -553,8 +568,8 @@ def is_any_frame():
             result = None
 
         f = self._is_cython_func(arg)
-        if f and not args and not kwargs:
-            return getattr(self, f)(), None
+        if f:
+            return getattr(self, f)(*args, **kwargs), None
```

Comment:
Is this equivalent? Wondering if there's any case where providing args / kwargs before would have routed the function to a different place

Reply:
It's not, but this is better :-). This fixes the bug in that was causing #19629 to fail.

The issue is subtle, but it has to do with a bug in numpy < 1.13 np.nan* functions. Numpy < 1.13 handles e.g. ``np.nanmin(pd_obj)`` incorrectly while it handles ``np.min(pd_obj)`` correctly. Numpy >= 1.13 handles both correctly. See #19753 for another issue regarding the same numpy problem.

Anyway, if ``f`` is a string, you *should* call ``getattr(self, f)(*args, **kwargs)``, so it was perhaps more luck than design that the previous version did work:-)